torch/_export/verifier.py

```diff
@@ -187,7 +187,7 @@ def _allowed_op_types() -> Tuple[Type[Any], ...]:
             )
 
             if not isinstance(op, _allowed_op_types()):
-                if op not in _allowed_builtin_ops() and op not in _allowed_torch_functions:
+                if op not in _allowed_builtin_ops() and op not in _allowed_torch_functions and not isinstance(op, torch._export.serde.serialize.CustomOpHandler):
```

Comment:
what's the error you saw? I'm confused by the change here.
```suggestion
                if op not in _allowed_builtin_ops() and op not in _allowed_torch_functions:
```

Reply:
then sounds like we need to change _allowed_op_types()?